sql/pg_scripts/ght-restore-pg

```diff
@@ -55,30 +55,52 @@ fi
 # Convert to full path
 dumpDir="`pwd`/$dumpDir"
 
-if [ ! -e $dumpDir/schema.sql ]; then
-  echo "Cannot find $dumpDir/schema.sql to create DB schema"
+pgDir=`echo $dumpDir/pg_scipts`
+
+if [ ! -e $pgDir/schema.sql ]; then
+  echo "Cannot find $pgDir/schema.sql to create DB schema"
   exit 1
 fi
 
 # 1. Create db schema
 echo "`date` Creating the DB schema"
-cat $dumpDir/schema.sql | $psql
+cat $pgDir/schema.sql | $psql
+
+count=`ls -1 $dumpDir/*.csv | wc -l`
+if [ $count != 0 ]; then
+   for f in $dumpDir/*.csv ; do
+     gzip $f
```

Comment:
Why do you compress the CSV files here only to decompress them while importing? The dump is almost 800GB, this will take a very significant amount of time.